srcdoc/configure/h2olog.mt

```diff
@@ -0,0 +1,85 @@
+? my $ctx = $main::context;
+? $_mt->wrapper_file("wrapper.mt", "Configure", "Using h2olog for Tracing")->(sub {
+
+<p>h2olog is a <a href="https://www.kernel.org/doc/html/latest/bpf/index.html">BPF</a> (<a href="https://www.kernel.org/doc/Documentation/networking/filter.txt">kernel doc</a>) backed HTTP request tracing tool for the <a href="https://github.com/h2o/h2o">H2O</a> server.
+h2olog can also be used to log <a href="https://en.wikipedia.org/wiki/QUIC">QUIC</a> events for <a href="https://en.wikipedia.org/wiki/Transport_layer">transport layer</a> observation.
+See <a href="#tracing-quic-events">Tracing QUIC events</a> for how.</p>
+
+<h2 id="installing-from-source">Installing from Source</h2>
+
+<p>See <a href="#requirements">requirements</a> for build prerequisites. If dependencies are satisfied, <code>-DWITH_H2OLOG=on</code> will be set and the <code>h2olog</code> target will be configured.</p>
```

Comment:
Maybe: If dependencies are satisfied, h2olog is built automatically. It is possible to manually turn on / off the build of h2log by using the <code>-DWITH_H2OLOG</code> option. This option takes either `ON` or `OFF` as the argument.